strongbox-data-service/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+    version="2.0">
+
+    <persistence-unit name="SBPersistenceUnit">
```

Comment:
What is the meaning of SB prefix? Will we have another persistent units? Suggest to have just "persistenceUnit" or so.